src/sources/gdal.jl

```diff
@@ -46,7 +46,7 @@ function Base.write(
     _gdalwrite(filename, correctedA, nbands; kw...)
 end
 function Base.write(
-    filename::AbstractString, ::Type{GDALfile}, A::AbstractRaster{T,3}, kw...
+    filename::AbstractString, ::Type{GDALfile}, A::AbstractRaster{T,3}; kw...
```

Comment:
This is a tiny bug fix by the way.

Reply:
yeah, noted. I think that path just doesn't get hit somehow